app/javascript/components/pages/Home.js

```diff
@@ -3,10 +3,32 @@ import blackCat from '../assets/black-cat.png'
 import { Button, Form, FormGroup, Input, Label, Row, Col } from 'reactstrap'
 
 class Home extends Component {
+  
```

Comment:
This section looks incomplete. Is your `console.log` on line 16 logging what you expect to see?